src/Symfony/Bundle/FrameworkBundle/Resources/config/identity_translator.xml

```diff
@@ -10,7 +10,10 @@
         <service id="Symfony\Component\Translation\TranslatorInterface" alias="translator" />
         <service id="Symfony\Contracts\Translation\TranslatorInterface" alias="translator" />
 
-        <service id="identity_translator" class="Symfony\Component\Translation\IdentityTranslator" />
+        <service id="identity_translator" class="Symfony\Component\Translation\IdentityTranslator">
+            <argument>null</argument>
```

Comment:
We need to drop this argument now so that our service definition is forward compatible.